webpack/global_index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { addGlobalFill } from 'foremanReact/components/common/Fill/GlobalFill';
+import { registerReducer } from 'foremanReact/common/MountingService';
+import API from 'foremanReact/API';
+import reducers from './ForemanRhCloudReducers';
+import InsightsHostDetailsTab from './InsightsHostDetailsTab';
+import { subscriptionCountListener } from './ForemanInventoryUpload/SubscriptionsPageExtension/SubscriptionsPageExtensionActions';
+import { foremanUrl } from './ForemanRhCloudHelpers';
+
+(async () => {
+  // Register reducers
+  Object.entries(reducers).forEach(([key, reducer]) =>
+    registerReducer(key, reducer)
+  );
+
+  // Register css
+  const {
+    data: { linkElement },
+  } = await API.get(foremanUrl('/foreman_rh_cloud/plugin_css_path'));
+  document
+    .getElementsByTagName('head')[0]
+    .insertAdjacentHTML('beforeend', linkElement);
+
+  const [prefix, mainPath, subPath] = window.location.pathname.split('/');
+  switch (mainPath) {
+    case 'subscriptions':
+      window.tfm.store.observeStore(
+        'katello.subscriptions.itemCount',
+        subscriptionCountListener
```

Comment:
what is the purpose of this?

Reply:
it was brought from the older global_js file